src/actinia_core/resources/common/process_chain.py

```diff
@@ -989,6 +991,9 @@ def _create_module_process(self, module_descr):
         if "verbose" in module_descr and module_descr["verbose"] is True:
             params.append("--v")
 
+        if "interface-description" in module_descr and module_descr["interface-description"] == "True":
```

Comment:
Just a question, why is it `"True"` here (a string), but for the `verbose` parameter above it is `True` (a boolean)?

Reply:
Very good hint, there was some process-chain confusion about that value. We found it thanks to your hint and fixed it. I changed above to a boolean